docs/source/audio_load.mdx

```diff
@@ -57,7 +57,41 @@ If you only want to load the underlying path to the audio dataset without decodi
 
 ## AudioFolder
 
-You can also load a dataset with an `AudioFolder` dataset builder. It does not require writing a custom dataloader, making it useful for quickly loading audio data.
+You can also load a dataset with an `AudioFolder` dataset builder. It does not require writing a custom dataloader, making it useful for quickly loading audio data. For classification tasks, your audio dataset structure should look like this:
+
+```
+folder/train/electronic/01.mp3
+folder/train/punk/01.mp3
```

Comment:
As @polinaeterna indicated, if you have a "test" subfolder, it will work too, so we might include that for both audio and image folder?

Reply:
yes, might be worth adding that 
```
folder/train/electronic/01.mp3
folder/train/punk/01.mp3
...
folder/test/electronic/09.mp3
folder/test/punk/09.mp3
```
would also work.

Reply:
lol there is actually a separate page in the documentation about how to structure a repo to include many splits. sorry for forgetting about it 🤦‍♀️ 
I think it definitely makes sense to put the link to it somewhere here:
https://huggingface.co/docs/datasets/v2.4.0/en/repository_structure#structure-your-repository